tests/network-setups/fabric/dev/Makefile

```diff
@@ -26,23 +29,39 @@ start-network2: .fabric-setup setup-cc
 .PHONY: start-interop-network1
 start-interop-network1:	setup-interop-cc start-network1 setup-interop-network1
 
+.PHONY: start-interop-network1-local
+start-interop-network1-local: setup-interop-cc-local start-network1 setup-interop-network1-local
+
 .PHONY:	start-interop-network2
 start-interop-network2: setup-interop-cc start-network2 setup-interop-network2
 
+.PHONY:	start-interop-network2-local
```

Comment:
Indentation issue. Here and couple of lines above and below as well (setup-interop-network2)

Reply:
Fixed.